test/services/sync/strategies/LDAPSyncerConsumer.integration.test.js

```diff
@@ -0,0 +1,153 @@
+const chai = require('chai');
```

Comment:
these probably could have been done as unittests

Reply:
we have unit tests where we mock repositories and integration tests which make the database changes.